core/src/test/scala/com/cloudera/livy/LivyTestFixture.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to Cloudera, Inc. under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Cloudera, Inc. licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.livy
+
+import org.scalatest.{Outcome, Suite}
+
+trait LivyTestFixture extends Suite with Logging {
```

Comment:
Can we rename it to LivyBaseUnitTestSuite?
I might add some other features to this class.
1. In the future I want to change all FlatSpec unit tests to use FunSpec. After that we can change this class to extend FunSuite and enforce FunSpec style on all unit tests.
2. Uploading logs to pastebin/hastebin when a unit test fail.